portfolio-project-matching-app/pages/index.js

```diff
@@ -13,6 +14,9 @@ export default function Home({ docList }) {
         <title>CS467 Portfolio Project</title>
       </Head>
       <DBList docList={docList} />
+      <Input type='text' name='text' placeholder='text'/>
+      <Input type='password' name='password' placeholder='password'/>
+      <Input type='email' name='password' placeholder='email'/>
```

Comment:
Glad to see I'm not the only one tinkering in `index.js`! I think it makes sense with how far along we are. I was making `test_componentName.js` Pages, but wasn't really sold on that concept. In most cases, you would just demo the component in the appropriate place in the app.